content/source/docs/extend/how-terraform-works.html.md

```diff
@@ -118,9 +118,9 @@ releases.hashicorp.com for newer acceptable provider versions and downloads them
 if available.
 
 This behavior only applies to providers whose _only_ acceptable versions are in
-`.terraform/plugins/<OS>_<ARCH>` (the automatic downloads directory); if any
-acceptable version of a given provider is installed elsewhere,
-`terraform init -upgrade` will not download a newer version of it.
+the correct subdirectories under `.terraform/plugins/` (the automatic downloads
```

Comment:
Isn't this `.terraform/providers` now? Am I misunderstanding the context of these changes?

Reply:
Nope - the new _install_ dir is `.terraform/providers`; we left `.terraform/plugins` alone since it was (widely) documented as the directory to drop provider binaries in: 
https://github.com/hashicorp/terraform/blob/23800438ab40890cac41440aa4fd6c798265fb71/command/meta_providers.go#L110-L113

I'll go check that other ref to OS_ARCH, thank you!

Reply:
Ah! Maybe "the automatic downloads directory" is now misleading, then?

Reply:
OH HRM that's a very good point - the directory referred to in this section IS `.providers`. 

arg sorry 🙃 I haven't looked closely at this in too long (and did not read this page closely enough)

Reply:
oh heck this whole page is very out of date
i have regrets